cli/npm/cache.rs

```diff
@@ -189,6 +195,16 @@ impl NpmCache {
     dist: &NpmPackageVersionDistInfo,
     registry_url: &Url,
   ) -> Result<(), AnyError> {
+    if self.no_npm {
```

Comment:
I think it might be better to put these were the unstable check is. Then you can also get better errors because here it only can display the resolved package name and version rather than the version requirement that the user specified.

Reply:
Done